tests/e2e/matmul/generate_e2e_matmul_tests.py

```diff
@@ -54,6 +54,13 @@ class CompilationInfoId(enum.Enum):
     SPIRVVectorizeNVIDIA = "SPIRVVectorizeNVIDIA"
 
 
+# Enumerates of supported transposed cases.
+@enum.unique
+class TransposeCase(enum.Enum):
+    NONE = "none"  # No transpose--canonical (M, K) x (K, N) -> (M, N)
+    B = "b"  # Transepose B matrix--(M, K) x (N, K) -> (M, N)
```

Comment:
The terminology in use throughout this test suite is

lhs for A
rhs for B
acc for C

Please keep consistent with that, ie. change the enum string from "b" to "rhs". Also in the comment.

Side note: I don't expect that we will ever transpose anything else than RHS here, and if we were to generalize, the term "transpose" would probably not scale to the kind of generalized matmuls that could arise. I would for now just do a boolean flag, `--transpose_rhs`, with no generalizability for now. Up to you.

Reply:
Fair point. I don't think I'll add support for other transpose cases anytime soon. So dropped that for now to opt in for simplicity.